Telemetry/CorePing.swift

```diff
@@ -57,6 +57,12 @@ class CorePing: TelemetryPing {
 
         let locale = NSBundle.mainBundle().preferredLocalizations.first!.stringByReplacingOccurrencesOfString("_", withString: "-")
 
+        let formatter = NSDateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd"
+        let date = formatter.stringFromDate(NSDate())
+
+        let timezoneOffset = NSTimeZone.localTimeZone().secondsFromGMT / 60
```

Comment:
Looks like it. I'm seeing -420, and -420/60 = -7 (where -8 would be PST). But I hadn't thought about it -- good observation!